evalml/data_checks/datetime_format_data_check.py

```diff
@@ -25,12 +25,12 @@ def validate(self, X, y):
         Returns:
             dict (DataCheckError): List with DataCheckErrors if unequal intervals are found in the datetime column.
 
-        Example:
+        Examples:
```

Comment:
Could just be personal opinion, but I think if we add this many examples (which is great!), it gets a little harder to read. Maybe what we need here is an extra line above each example to better understand what each example is trying to convey? From a quick glance, it's a bit hard to gauge the purpose of each example 😬

Reply:
Sure thing, so maybe a comment above a test briefly explaining what it does?

Reply:
Exactly!